config/ecs/container-insights/otel-agent-default-config.yaml

```diff
@@ -0,0 +1,46 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:55680
+      http:
+        endpoint: localhost:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+  awsecscontainermetrics:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+  batch/metrics:
+    timeout: 60s
+
+exporters:
+  awsxray:
+  awsemf/application:
+    log_group_name: aws/ecs/containerinsights/{aws.ecs.cluster.name}/application/metrics
+    log_stream_name: { aws.ecs.task.id }
+  awsemf/performance:
+    log_group_name: /aws/ecs/containerinsights/{aws.ecs.cluster.name}/performance
+    log_stream_name: { aws.ecs.task.id }
```

Comment:
For container insights metrics... there is a lot more that needs to be done. You must select the right metrics, change their names, add the resource labels as dimensions, and etc.

@hossain-rayhan Can help with this

Reply:
@JohnWu20 can you copy the ECS container metrics part from the shared doc. Then I can add more comments.